public/wp-content/themes/newsup-child/functions.php

```diff
@@ -185,13 +185,22 @@ function newsup_front_page_banner_section()
                                                       <?php endif; ?>
                                                       </a>
                                                   </div>
-
+                                                  <!-- POST FEATURE ITEM -->
                                                   <article class="bottom">
                                                           <span class="post-form"><i class="fa fa-camera"></i></span>
                                                           <div class="mg-blog-category"> <?php newsup_post_categories(); ?> </div>
                                                           <h1 class="title"> <a href="<?php the_permalink(); ?>"><?php the_title(); ?></a></h1>
-                                                          <?php newsup_post_meta(); ?>
```

Comment:
This function makes use of a template in newsup internal hooks. Let's just overwrite it by copy pasting because otherwise it would be a lot of work.